remix.init/index.js

```diff
@@ -72,7 +72,7 @@ async function updatePackageJsonForEdge(directory) {
     dependencies: {
       ...dependencies,
       "@netlify/edge-functions": "^2.0.0",
-      "@netlify/remix-edge-adapter": "1.0.0",
+      "@netlify/remix-edge-adapter": "1.1.0",
```

Comment:
This brings in the new default Remix configuration for the Netlify Edge Functions template, i.e. 

```js
future: {
  v2_routeConvention: true, 
}
```